### DISCUSSION TITLE
Convert mysql to the others.

### DISCUSSION THREAD

**Discussion by bbannon** [craigk5n/webcalendar#267] [Ideas]
Craig,
Instead of maintaining a table/update-*.sql for each db, why not modify mysql/maria.sql on the fly? It only takes about 25-30 lines of code to convert all the others. That's total, not each I had done it before the lightning strike, but didn't get a chance to upload it. Haven't had time to redo it.
Bruce

**Comment by craigk5n** [comment]
Sounds like an interesting idea.  I also have no idea how accurate the various SQL files are since I usually only test mysql and sqlite.

**Reply by bbannon** [reply]
Right now I only have a tablet so can't do much of anything. 
You might try setting up Decker images for the dbs.

**Comment by craigk5n** [comment]
I setup another docker example to test out PHP8.  I'll plan on doing something similar for postgres.

Have you tried any of the cloud-based PHP IDEs like [Eclipse Che](https://www.eclipse.org/che/)?

**Reply by bbannon** [reply]
Never heard of Eclipse Che. What does it do?  I went to the site, but can't read it on this tiny screen.

If set up dockers for the various dbs, is there a way to only download the one the user needs?
Also, what about only download the translation txt file based on the user browser language?

Once upon a time, I was going to try to move English.txt into the code to cut down on missing English phrases.
Any thoughts on that?
